.github/workflows/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: CI
+on: 
+  pull_request:
+  push:
+    
+jobs:
+  jpo-security-svcs:
+   runs-on: ubuntu-latest
+   container:
+     image: openjdk:17-jdk-slim-buster
```

Comment:
Probably want to use jdk 21.

Reply:
This has been changed to maven:3.8-eclipse-temurin-21-alpine